### PR TITLE
Bug 1113605 Share a contact

### PR DESCRIPTION
**Pull request by jmcanterafonseca**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[jmcanterafonseca Jose M. Cantera (jmcanterafonseca)](https://github.com/jmcanterafonseca) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=1394af810504)

File: apps/communications/contacts/js/views/details.js [hunk 1]
```diff
@@ -624,6 +631,36 @@ contacts.Details = (function() {
     cover.dataset.imgHash = '';
   };
 
+  var shareContact = function cd_shareContact() {
+    if (fb.isFbContact(contactData)) {
+      alert(_('facebook-export-forbidden'));
```
Comment (arcturus) [inline apps/communications/contacts/js/views/details.js]: Please remove this `alert`

Comment (arcturus) [inline apps/communications/contacts/js/views/details.js]: Actually, we shouldn't need to recheck isnt? This is enabled just for non fb contacts

**Comment by try-server-hook** [comment]
[jmcanterafonseca Jose M. Cantera (jmcanterafonseca)](https://github.com/jmcanterafonseca) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=043cb7b2a389)

File: apps/communications/contacts/js/views/details.js [hunk 2]
```diff
@@ -624,6 +631,31 @@ contacts.Details = (function() {
...
+          data: {
+            type: 'text/vcard',
+            number: 1,
+            blobs: [vcardBlob],
+            filenames: [VcardFilename(contactData)]
+          }
+        });
+      }, {
+            // The MIME of the blob should be this for some MMS gateways
+            type: 'text/x-vcard'
```
Comment (arcturus) [inline apps/communications/contacts/js/views/details.js]: nit: indentation (looks weird to me)

**Comment by try-server-hook** [comment]
[jmcanterafonseca Jose M. Cantera (jmcanterafonseca)](https://github.com/jmcanterafonseca) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ce5cae9d2fe3)

**Comment by try-server-hook** [comment]
[jmcanterafonseca Jose M. Cantera (jmcanterafonseca)](https://github.com/jmcanterafonseca) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=36d6897ba5bf)

**Pull request merged**